proto/poktroll/migration/tx.proto

```diff
@@ -111,3 +113,24 @@ message MsgClaimMorseAccountResponse {
   int64 claimed_at_height = 3;
 }
 
+// TODO_UPNEXT(@bryanchriswhite, #1034):
+// - Refactor to consistent naming convention
+// - Add comments describing the purpose of each message/field
+
+message MsgClaimMorseApplication {
+  option (cosmos.msg.v1.signer) = "shannonDestAddress";
+  string                   shannonDestAddress = 1;
+  string                   morseSrcAddress    = 2;
+  string                   morseSignature     = 3;
+  cosmos.base.v1beta1.Coin stake              = 4 [(gogoproto.nullable) = false];
+  string                   serviceConfig      = 5;
```

Comment:
1. We agree on no embedding `MsgStakeApplication` - aligned
2. I'm expecting to see a `repeated poktroll.shared.ApplicationServiceConfig services = 3;` here, similar to 
`proto/poktroll/application/tx.proto`, but tha'ts not the case.

Can you provide details (comments) on the plan or potentially tend to the changes here?

Reply:
As discussed, and reflected in https://github.com/pokt-network/poktroll/pull/1082/files#r1980342594, leaving this as a non-repeating field.

Reply:
Sounds, just #PUC